nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/indexing/conditions/Conditions.java

```diff
@@ -24,6 +24,29 @@
 
 public class Conditions {
 
+    private Conditions() {}
+
+    public enum ConditionMode {
+        EPSILON_EQUALS, //0
+        EPSILON_NOT_EQUALS, //1
+        LESS_THAN, //2
+        GREATER_THAN, //3
+        LESS_THAN_OR_EQUAL, //4
+        GREATER_THAN_OR_EQUAL, //5
+        ABS_LESS_THAN, //6
+        ABS_GREATER_THAN, //7
+        IS_INFINITE, //8
+        IS_NAN, //9
+        ABS_EQUALS, //10
+        NOT_EQUALS, //11
+        ABS_GREATER_OR_EQUAL, //12
+        ABS_LESS_THAN_OR_EQUAL, //13
+        IS_FINITE, //14
+        NOT_FINITE, //15
+
+    }
```

Comment:
Would be nice to make that fixed so resorting those wouldn't result in changes. 

Also it would be very good to use them as Constants in all the places where we need to return a `conditionNum`